sklearn/linear_model/tests/ransac_iterations.py

```diff
@@ -0,0 +1,20 @@
+from sklearn import linear_model
+from sklearn.linear_model.ransac import _dynamic_max_trials
+import numpy as np
+
+X =np.linspace(0,100,100)[:,None]
+print X.shape
+Y = 2*X
+Y[80:] = np.random.rand(20,1)
+Y[:80] += np.random.rand(80,1)
+
+for i in range(10):
+    clt_ransac = linear_model.RANSACRegressor(linear_model.LinearRegression(),max_trials=100)
+    clt_ransac.fit(X, Y)
+    yhat_ransac = clt_ransac.predict(X)
+    inlier_mask = clt_ransac.inlier_mask_
+    print '*************************'
+    print 'actual     trials:' , clt_ransac.n_trials_
```

Comment:
I see you have a test for this (test_ransac_max_trials). but there are only 3 outliers and you run the test once. if you up the outliers to 40 (~10%), don't set random_state to zero, it will run 4 iterations once every 100ish runs. but that's ok.